package.json

```
@@ -0,0 +1,10 @@
+{
+  "devDependencies": {
+    "vitepress": "^1.6.3"
+  },
+  "scripts": {
+    "docs:dev": "vitepress dev",
+    "docs:build": "vitepress build",
+    "docs:preview": "vitepress preview"
```

Did you test that these commands work (e.g. `npm run docs:dev`)? The [docs](https://vitepress.dev/guide/getting-started#up-and-running) say they should be:
```suggestion
    "docs:dev": "vitepress dev docs",
    "docs:build": "vitepress build docs",
    "docs:preview": "vitepress preview docs"
```

---

yes they do run.  when @georgezgeorgez requested that I move the markdown docs from /docs to root I had to modify this.

---

yeah that's just a path
it might be more clear if the following works `vitepress dev .` but if it works as is, no problem